app/assets/javascripts/components/overview/course_stats.jsx

```diff
@@ -20,6 +47,8 @@ const CourseStats = ({ course }) => {
         <small>{I18n.t('metrics.word_count')}</small>
       </div>
     );
+  } else if (isWikidata) {
```

Comment:
This seems like an odd move. Can we rephrase this into an affirmative `if/else` statement? For example:
```js
else if (!isWikidata) {
```